cypress/e2e/WebInterface/Smoke Tests/EditMeasure.cy.ts

```diff
@@ -39,13 +39,14 @@ describe('Edit Measure', () => {
             }).then((response) => {
                 expect(response.status).to.eql(201)
                 cy.writeFile('cypress/fixtures/measureId', response.body.id)
+                cy.writeFile('cypress/fixtures/measureSetId', response.body.measureSetId)
```

Comment:
This might actually be a "two for one". The existing test was failing during the `after` hook because it didn't have the correct measureSetId value during the measure deletion.